packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -514,7 +537,17 @@ class Navigator extends StatefulWidget {
   /// A list of observers for this navigator.
   final List<NavigatorObserver> observers;
 
-  /// The default name for the initial route.
+  /// The separator used to distinguish parts of a route.
+  ///
+  /// See [initialRoute].
+  static const String routeSeparator = '/';
```

Comment:
If we're going to have a constant instead of a convention, then our examples and framework probably need a survey, to see where '/' is assumed.